assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -78,7 +78,7 @@ function PagePreview({ index, gridRef, ...props }) {
   }));
   const page = pages[index];
   const { backgroundColor } = page;
-  const { width: thumbWidth, height: thumbHeight, isActive } = props;
+  const { width: thumbWidth, height: thumbHeight, isActive, ariaLabel } = props;
```

Comment:
Since there's no need to keep it in `props` (I guess `<Page />` doesn't do anything about the `ariaLabel` prop), you can destruct it out above in the function definition:

```js
function PagePreview({ index, gridRef, ariaLabel, ...props }) {
```

Alternatively, you can make the parent component set `aria-label` rather than `ariaLabel` and it should probably work without any other changes.

Reply:
yea absolutely. sounds good. will update

Reply:
Looking further into this, you can simply [rename `ariaLabel` to `aria-label`](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/canvas/carousel/index.js#L418) in the parent `<Carousel />` component. And then just let it pass through `...props` here without explicitly mentioning it.

With the renaming of `ariaLabel` to `aria-label` in `<Carousel />`, you can then completely delete the components  
`<CompactIndicatorWithRef />` and `<CompactIndicator />` in [the file `compactIndicator.js`](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/canvas/carousel/compactIndicator.js#L30-L40) altogether and just make `Indicator` default export directly in that file (maybe rename it to `CompactIndicator` so it fits with the filename).

That'll reduce the complexity slightly of this file and significantly of `compactIndicator.js` - all the forward-ref and proptype-stuff in that one can just go out the door!

Reply:
ooh yea. very nice